set-versions

```diff
@@ -38,7 +38,7 @@ for project in $(find . -name project.clj -exec dirname '{}' \;); do
 
   cljproj=$(basename $project)
 
-  if [ "$cljproj" == "Donkey" ]; then
+  if [ "$cljproj" == "Terrain" ]; then
```

Comment:
@johnworth could probably explain this better than I can, but these version numbers were used for RPM versions before we went with Docker, and the version got bumped for every QA drop, so they outpaced the DE version. `5.0.0` is just where these versions got to at DE v2.0 when we switched to Docker. Not sure why these versions haven't been updated since then.